File: 485_MaxConsecutiveOnes.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int maxTillNow = 0;
        for (int i=0; i<nums.length; i++) {
            if (nums[i] == 1) {
                maxTillNow++;
            } else {
                maxTillNow = 0;
            }
            max = Math.max(max, maxTillNow);
        }
        return max;
    }
}

File: 485_MaxConsecutiveOnes.c
int findMaxConsecutiveOnes(int* nums, int numsSize){
    int count = 0, maxOnes = 0;
    for (int i=0; i<numsSize; i++) {
        if (nums[i] == 1) {
            count++;
        } else {
            count = 0;
        }
        maxOnes = fmax(maxOnes, count);
    }
    return maxOnes;
}
